core/templates/pages/contributor-dashboard-page/contributor-badge/contributor-badge.component.html

```diff
@@ -0,0 +1,552 @@
+  }
+  .mobile-stats-row {
+    display: flex;
+    margin-bottom: 5px;
+  }
+  .mobile-stats-name {
+    width: 50%;
+  }
+  .mobile-stats-data-container:not(:last-child) {
+    border-bottom: 1px solid #bbb;
+  }
+  .stats-column-name-container {
+    margin-left: auto;
+    margin-right: auto;
+    width: 75%;
+  }
+  .stats-data {
+    text-align: center;
+  }
+  .stats-welcome-text-container {
+    color: #00645C;
+    display: inline-block;
+    font-family: "Roboto", Arial, sans-serif;
+    font-size: 18px;
+    height: 30%;
+    width: 60%;
+  }
+  .stats-contribution-type-container {
+    display: inline-block;
+    width: 40%;
+    vertical-align: top;
+  }
+  .stats-contribution-type-label {
+    font-size: 18px;
+  }
+  .stats-container {
+    margin-left: 50px;
+  }
+  .list-view-item {
+    background: #fff;
+    box-shadow: none;
+    border: 1px solid #000;
+    margin-top: 10px;
+    margin-bottom: 25px;
+  }
+  .dashboard-table {
+    font-size: 1.2vw;
+    width: 100%;
+    margin-top: 40px;
+  }
+  .dashboard-table .dashboard-table-headings {
+    border-bottom: 2px solid black;
+    padding-bottom: 0.7em;
+    text-align: center;
+  }
+
+  .dashboard-table .dashboard-table-headings p {
+    display: inline;
+  }
+  .dashboard-table #dashboard-table-headings-details {
+    display: inline-block;
+    text-align: left;
+    width: 100%;
+  }
+  .stats-row {
+    height: 50px;
+    text-align: left;
+    vertical-align: baseline;
+  }
+  .stats-row td {
+    padding-top: 15px;
+  }
+  .stats-row:not(:last-child) {
+    border-bottom: 1px solid #bbb;
+  }
+  .oppia-stats-type-selector-container {
+    background-color: #fff;
+    border-radius: 4px;
+    box-shadow: 0 0 4px rgba(0, 0, 0, 0.25);
+    cursor: pointer;
+    justify-content: space-between;
+    padding: 10px 20px;
+    position: relative;
+    vertical-align: top;
+  }
+  .oppia-stats-type-selector-inner-container {
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+    font-size: 16px;
+  }
+
+  .oppia-stats-type-selector-inner-container-arrow {
+    color: #009688;
+    cursor: pointer;
+    padding-top: 3px;
+  }
+  .oppia-stats-type-selector-dropdown-container {
+    background-color: #fff;
+    border-radius: 4px;
+    box-shadow: 0 0 4px rgba(0, 0, 0, 0.25);
+    height: auto;
+    left: 0;
+    overflow-y: scroll;
+    padding: 8px 12px;
+    position: absolute;
+    right: 0;
+    top: 100%;
+    z-index: 1;
+  }
+  .oppia-stats-type-selector-dropdown-container::-webkit-scrollbar {
+    width: 7px;
+  }
+  .oppia-stats-type-selector-dropdown-container::-webkit-scrollbar-track {
+    border-radius: 5px;
+    box-shadow: inset 0 0 3px #808080;
+  }
+  .oppia-stats-type-selector-dropdown-container::-webkit-scrollbar-thumb {
+    background: #808080;
+    border-radius: 5px;
+    transition: background 0.5s ease;
+  }
+  .oppia-stats-type-selector-dropdown-container::-webkit-scrollbar-thumb:hover {
+    background: black;
+  }
```

Comment:
Should we handle other than WebKit scrollbars too?

Reply:
Hmm, did you check that the UI works correctly in non-webkit browsers?

Reply:
I can not test it because I do not have a non-webkit browser. But should not be still an issue since we are already do this way in the contributor dashboard?

Reply:
Can you ask someone for testing? Or can you download Firefox for testing?

Reply:
https://user-images.githubusercontent.com/41409951/203468683-e63721e3-5f5d-4e37-8f37-f4b147b420f4.mov

It seems fine in firefox.